corehq/messaging/management/commands/queue_schedule_instances.py

```diff
@@ -0,0 +1,85 @@
+from corehq.apps.domain_migration_flags.api import any_migrations_in_progress
+from corehq.messaging.scheduling.scheduling_partitioned.dbaccessors import (
+    get_active_schedule_instance_ids,
+    get_active_case_schedule_instance_ids,
+)
+from corehq.messaging.scheduling.scheduling_partitioned.models import (
+    AlertScheduleInstance,
+    TimedScheduleInstance,
+    CaseAlertScheduleInstance,
+    CaseTimedScheduleInstance,
+)
+from corehq.messaging.scheduling.tasks import (
+    handle_alert_schedule_instance,
+    handle_timed_schedule_instance,
+    handle_case_alert_schedule_instance,
+    handle_case_timed_schedule_instance,
+)
+from corehq.sql_db.util import handle_connection_failure, get_default_and_partitioned_db_aliases
+from corehq.toggles import DATA_MIGRATION
+from datetime import datetime
+from dimagi.utils.couch.cache.cache_core import get_redis_client
+from dimagi.utils.logging import notify_exception
+from django.core.management.base import BaseCommand
+from time import sleep
+
+
+def skip_domain(domain):
+    return DATA_MIGRATION.enabled(domain) or any_migrations_in_progress(domain)
+
+
+class Command(BaseCommand):
+    args = ""
```

Comment:
`args` was removed in Django 1.10. `add_arguments` function is the only way to specify args now